packages/react-components/react-combobox/src/components/Combobox/useCombobox.ts

```diff
@@ -55,7 +55,7 @@ export const useCombobox_unstable = (props: ComboboxProps, ref: React.Ref<HTMLBu
   const popperOptions = {
```

Comment:
(nit) rename `popperOptions` to `positioningOptions`

Reply:
makes sense done in [d5505bf](https://github.com/microsoft/fluentui/pull/22586/commits/d5505bf0a554897c7405c385726c8681bd0268bc)